### ISSUE TITLE
Visual 'volume' amplifier for oscilloscope.

### ISSUE THREAD

**Issue by Mid-The-Modder** [Mid-The-Modder/Midbox-Source#54]
I want to add a slider to the oscilloscope that acts like as if the song's waveform was much quieter/louder. Should be a matter of placing a slider underneath it, saving it as a preference, and letting that modify how the oscilloscope scales the waveform.
